src/Common/tests/Common.Tests.csproj

```diff
@@ -55,10 +58,14 @@
     <Compile Include="..\src\System\Net\HttpKnownHeaderNames.TryGetHeaderName.cs">
       <Link>Common\System\Net\HttpKnownHeaderNames.TryGetHeaderName.cs</Link>
     </Compile>
-    <Compile Include="Tests\System\IO\PathInternal_Unix_Tests.cs" />
+    <Compile Include="$(CommonTestPath)\System\AssertExtensions.cs">
+      <Link>CommonTest\System\AssertExtensions.cs</Link>
+    </Compile>
     <Compile Include="Tests\System\CharArrayHelpersTests.cs" />
-    <Compile Include="Tests\System\IO\RowConfigReaderTests.cs" />
     <Compile Include="Tests\System\StringExtensions.Tests.cs" />
+    <Compile Include="Tests\System\Collections\Generic\ArrayBuilderTests.cs" />
+    <Compile Include="Tests\System\IO\PathInternal_Unix_Tests.cs" />
+    <Compile Include="Tests\System\IO\RowConfigReaderTests.cs" />
```

Comment:
Re-organized the `StringExtensions.Tests.cs` to be next to the `CharArrayHelpersTests.cs` here